test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -41,39 +44,63 @@ public BatchTests(HttpIntegrationTestFixture fixture)
             _client = fixture.TestFhirClient;
         }
 
-        [Fact]
+        [SkippableFact]
```

Comment:
Should this be a theory that tests both sequential and parallel logic?

Reply:
Done :)